packages/@react-aria/overlays/src/useOverlayPosition.ts

```diff
@@ -12,13 +12,13 @@
 
 import {calculatePosition, PositionResult} from './calculatePosition';
 import {HTMLAttributes, RefObject, useEffect, useState} from 'react';
-import {PlacementAxis} from '@react-types/overlays';
+import {Placement, PlacementAxis} from '@react-types/overlays';
 import {useLocale} from '@react-aria/i18n';
 
-export type Placement = 'bottom' | 'bottom left' | 'bottom right' | 'bottom start' | 'bottom end' |
-    'top' | 'top left' | 'top right' | 'top start' | 'top end' |
-    'left' | 'left top' | 'left bottom' | 'start' | 'start top' | 'start bottom' |
-    'right' | 'right top' | 'right bottom' | 'end' | 'end top' | 'end bottom';
+// export type Placement = 'bottom' | 'bottom left' | 'bottom right' | 'bottom start' | 'bottom end' |
```

Comment:
why isn't this deleted?

Reply:
It was moved to a shared location. See import above.